ddtrace/contrib/openai/patch.py

```diff
@@ -34,11 +34,14 @@
     "openai",
     {
         "logs_enabled": asbool(os.getenv("DD_OPENAI_LOGS_ENABLED", False)),
+        "llmobs_enabled": asbool(os.getenv("DD_OPENAI_LLMOBS_ENABLED", False)),
         "metrics_enabled": asbool(os.getenv("DD_OPENAI_METRICS_ENABLED", True)),
         "span_prompt_completion_sample_rate": float(os.getenv("DD_OPENAI_SPAN_PROMPT_COMPLETION_SAMPLE_RATE", 1.0)),
+        "llmobs_prompt_completion_sample_rate": float(os.getenv("DD_OPENAI_LLMOBS_PROMPT_COMPLETION_SAMPLE_RATE", 1.0)),
         "log_prompt_completion_sample_rate": float(os.getenv("DD_OPENAI_LOG_PROMPT_COMPLETION_SAMPLE_RATE", 0.1)),
         "span_char_limit": int(os.getenv("DD_OPENAI_SPAN_CHAR_LIMIT", 128)),
         "_api_key": os.getenv("DD_API_KEY"),
+        "_app_key": os.getenv("DD_APP_KEY"),
```

Comment:
These configurations are getting a bit messy. Can/should we use envier to organize them?

Reply:
We're planning on removing logs payloads entirely in the near future, so things like `logs_enabled`, `log_prompt_completion_sample_rate` will be removed and make this less messy - I don't think we need to reorganized at this time. 

If at that point (once we remove logs) is still too messy, we can look at cleaning up.